src/sentry/tsdb/base.py

```diff
@@ -20,6 +20,7 @@ class TSDBModel(Enum):
     project = 1
     group = 4
     release = 7
+    issue_platform = 8
```

Comment:
I think we need to have these in sync

https://github.com/getsentry/sentry/pull/42904/files#diff-7a3b68e1f1e08044d4de3d8898c14d421a3e314f8901d0dd5d640b244a8c14aa

Reply:
Oh yeah I'm just gonna close this PR since Gilbert's PR duplicates it. We talked about it offline, just forgot to update here.